packages/fxa-payments-server/src/components/fields.tsx

```diff
@@ -130,6 +133,8 @@ export const Input = (props: InputProps) => {
     [name, validator, onValidate, required]
   );
 
+  const onChangeWrapper = (ev: any) => onChange(ev);
```

Comment:
Doesn't look like this wrapper does anything? Should also be defined using `React.useCallback` so as not to be re-created on every component render.

Reply:
This is to keep the typescript checker happy. It doesn't like assigning a function with two arguments to `onChange`